.travis.yml

```diff
@@ -1,27 +1,37 @@
 ---
+dist: bionic
```

Comment:
travis is using xenial by default which is quite old and have python 3.5 on system level
here I'm using bionic where we can use python 3.6 with `system_site_packages`
unfortunately python 3.7 is not available with `system_site_packages` option which is required to run `molecule prepare` step